software/twenty.yml

```
@@ -0,0 +1,12 @@
+name: Twenty
+website_url: "https://twenty.com"
+source_code_url: "https://github.com/twentyhq/twenty"
+description: A modern CRM offering the flexibility of open source, advanced features, and a sleek design.
+licenses:
+  - AGPL-3.0
+platforms:
+  - TypeScript
```

> https://docs.twenty.com/start/self-hosting
Right now, Docker containers are the only hosting option available

```suggestion
platforms:
  - Docker
```

---

@Ki-er Thank you for reviewing the PR! I've made the suggested changes.